modules/gapi/include/opencv2/gapi/streaming/gstreamer/gstreamerpipeline.hpp

```diff
@@ -19,15 +19,19 @@ namespace gapi {
 namespace wip {
 namespace gst {
 
-class GAPI_EXPORTS GStreamerPipeline
+class GAPI_EXPORTS_W GStreamerPipeline
 {
 public:
     class Priv;
 
-    explicit GStreamerPipeline(const std::string& pipeline);
+    GAPI_WRAP explicit GStreamerPipeline(const std::string& pipeline);
     IStreamSource::Ptr getStreamingSource(const std::string& appsinkName,
                                           const GStreamerSource::OutputType outputType =
                                               GStreamerSource::OutputType::MAT);
+    // NB: Function for using from python
```

Comment:
In fact it's  class `method`, I wouldn't put such obvious comment, since you mark it as `GAPI_WRAP`

Reply:
OK, removed